t/e2e/services/generic_api_test.go

```diff
@@ -0,0 +1,86 @@
+package services
```

Comment:
This file only defines a suite but it does not contain any tests. Should it have `_test` prefix in the name?

Reply:
I can't find any example on the current test suite, you mean a **file name** like `_test_basic_api_test.go`?

Or this review comment is about package name so it's moved to something like `services_test`?